Makefile

```diff
@@ -0,0 +1,39 @@
+MAKEFLAGS += -j2
+default: help
+
+help:
+	@echo "Build Container"
+	@echo
+	@echo "  - make build"
+	@echo
+	@echo "Install ramalama"
+	@echo
+	@echo "  - make install"
+	@echo
+	@echo "Test ramalama"
+	@echo
+	@echo "  - make test"
+	@echo
+	@echo "Clean the repository"
+	@echo
+	@echo "  - make test"
```

Comment:
Typo "make clean", if this goes green before you get to it, I'll merge it anyway and catch in a follow on PR